tests/fixtures/findings-to-issues/disposition.phase1.yaml

```
@@ -0,0 +1,3 @@
+findings_disposition:
+  - { file: "src/auth.ts", line: 88, disposition: APPLIED }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)